### PR TITLE
Windows Forms getting started update for .NET 6

### PR DESCRIPTION
**Pull request by adegeo**
- Add new VS 2022 images
- Move images to versioned folders
- Update article to cover both VS2019/VS2022 and .NET 5/.NET 6
- Convert inline code to snippet ref

[Internal preview 5.0](https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/get-started/create-app-visual-studio?view=netdesktop-5.0&branch=pr-en-us-1215)
[Internal preview 6.0](https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/get-started/create-app-visual-studio?view=netdesktop-6.0&branch=pr-en-us-1215)

#### TODO

- [x] Check on docfx versioning settings

### CODE REVIEW

File: dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md [hunk 1]
```diff
@@ -25,18 +25,32 @@ In this tutorial, you learn how to:
...
+
+:::moniker range="netdesktop-6.0"
+
+- [Visual Studio 2022 version 17.0 or later versions](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&0utm_content=download+vs2022+desktopguide+winforms)
+  - Select the [.NET desktop development workload](/visualstudio/install/modify-visual-studio?view=vs-2019&preserve-view=true#modify-workloads)
+  - Select the [.NET 6 individual component](/visualstudio/install/modify-visual-studio?view=vs-2019&preserve-view=true#modify-individual-components)
+
+:::moniker-end
+
 ## Create a WinForms app
```
Comment (RussKie) [inline dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md:44]: Personally I find it quite unsettling when "Windows Forms" and "WinForms" are constantly interchanged. If possible can we use "Windows Forms" everywhere?

**Review by RussKie — COMMENTED** [review]
👍

**Comment by adegeo** [comment]
@tdykstra please review, thanks!

**Review by tdykstra — APPROVED** [review]
Looks great, a couple of contributor guide recommendations for you to consider.

File: dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md [hunk 2]
```diff
@@ -1,42 +1,56 @@
...
 
 The first step to creating a new app is opening Visual Studio and generating the app from a template.
 
+:::moniker range="netdesktop-5.0"
+
 01. Open Visual Studio.
 01. Select **Create a new project**.
 
-    :::image type="content" source="media/create-app-visual-studio/vs-create-new-project.png" alt-text="Create a new Windows Forms project in Visual Studio 2019 for .NET":::
+    :::image type="content" source="media/create-app-visual-studio/netdesktop-5.0/vs-create-new-project.png" alt-text="Create a new Windows Forms project in Visual Studio 2019 for .NET":::
```
[the remark this answers is not in the document]
Comment (adegeo) [reply dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md:53]: I just checked on a fresh VM, dark mode is the suggested mode at start.

File: dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md [hunk 3]
```diff
@@ -45,21 +59,63 @@ The first step to creating a new app is opening Visual Studio and generating the
...
+    :::image type="content" source="media/create-app-visual-studio/netdesktop-5.0/vs-config-new-project.png" alt-text="Configure new Windows Forms project in Visual Studio 2019 for .NET":::
+
+:::moniker-end
+
+:::moniker range="netdesktop-6.0"
+
+01. Open Visual Studio.
+01. Select **Create a new project**.
+
+    :::image type="content" source="media/create-app-visual-studio/netdesktop-6.0/vs-create-new-project.png" alt-text="Create a new Windows Forms project in Visual Studio 2022 for .NET":::
```
[the remark this answers is not in the document]
Comment (adegeo) [reply dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md:77]:
```suggestion
    :::image type="content" source="media/create-app-visual-studio/netdesktop-6.0/vs-create-new-project.png" alt-text="Create a new Windows Forms project in Visual Studio 2022 for .NET.":::
```

**Pull request merged**